src/main/java/frc/robot/subsystems/Tank.java

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import com.ctre.phoenix.motorcontrol.can.BaseMotorController; 
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX; // internal Falcon motors
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+
+import edu.wpi.first.wpilibj.XboxController; // controller input library
+
+import static frc.robot.Constants.TankConstants.*;
+
+public class Tank extends SubsystemBase {
+    
+    private final WPI_TalonSRX left = new WPI_TalonSRX​(LEFT_MAIN); // left motor
```

Comment:
You seem to have [zero-width spaces](https://en.wikipedia.org/wiki/Zero-width_space) `ZWSP` sprinkled among your `WPI_TalonSRX` constructor calls?

![image](https://user-images.githubusercontent.com/60120929/204692382-e9aac51f-b275-47b3-b93f-f5378721ebd4.png)

Removing these should fix the build errors (I think VSCode should render these characters as boxes with `ZWSP` inside them? If not it may be a setting you have to enable). 

Also, I'd recommend using `MotorUtil.createTalonSRX()` for easier global configuration.